Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Search.hs

```diff
@@ -481,15 +481,16 @@ selectDriversAndMatchFarePolicies merchant merchantOpCityId mbDistance fromLocat
             mRadiusStep = Nothing,
             isRental = False,
             isInterCity = False,
+            rideFare = Nothing,
             ..
           }
-  (driverPoolNotOnRide, _) <- calculateDriverPool calculateDriverPoolReq
+  (driverPoolNotOnRide, _) <- calculateDriverPool calculateDriverPoolReq transporterConfig.prepaidSubscriptionThreshold
```

Comment:
but `transporterConfig` already included in `calculateDriverPoolReq`, why pass it separately?